dao/mongodb.rya/src/main/java/org/apache/rya/mongodb/dao/SimpleMongoDBStorageStrategy.java

```diff
@@ -53,8 +54,11 @@
     public static final String OBJECT_TYPE_VALUE = XMLSchema.ANYURI.stringValue();
     public static final String CONTEXT = "context";
     public static final String PREDICATE = "predicate";
-    public static final String OBJECT = "object";
+    public static final String PREDICATE_HASH = "predicate_hash";
+    public static final String OBJECT = "object_original";
```

Comment:
@pujav65 I'm concerned about index size.  please hash everything.  If you want another ticket for "please hash everything" I'm fine with that, but let's knock that out while @isper3at is cleaning this stuff up.  Key thing with mongo is to get the index to fit in memory, so lets do that.